pkg/plugins/files.go

```diff
@@ -38,5 +46,15 @@ func writeFile(file schema.File, fs vfs.FS) error {
 		return err
 
 	}
+
+	if file.OwnerString != "" {
```

Comment:
Interesting I was thinking on changes for File and Directory that could be similar. I believe that having Owner and Group IDs as integers makes it hard to distinguish behavior when they are set or unset in the yaml (if unset I'd assume the user and group are the ones from the user running yip)